_posts/2023-06-17-call-for-venue-proposal-2024.md

```diff
@@ -70,15 +61,14 @@ We need some information about your city, about the venue, the hotel, and travel
 	* Location
 	* Transportation options (how do people get there from the hotel?)
 	* Attendance capacity
-	* Cost of function space, catering, and audio-visual services. Sprints are often hosted by local companies as sponsorship of the event, which is preferable, as it saves the cost of renting a venue.
+	* Information on local companies that might host the sprints at no cost.
```

Comment:
Might be worth a note and I'm not in-love with this wording "if sprints aren't economically feasible to host as part of the same venue..."

Reply:
```suggestion
	* Information on local companies that might be willing to host the sprints at no cost.
```